docs/source/developers/guide/step_by_step/pr_and_github.rst

```diff
@@ -108,40 +108,44 @@ and linters and then go ahead and commit the changes.
    amend, delete, revise, etc. See https://git-scm.com/docs for more
    information.
 
-   Until you make the Pull Request, nothing is visible on the Arrow
+   Until you make the pull request, nothing is visible on the Arrow
    repository and you are free to experiment.
 
-If all is set, you can make the Pull Request!
+If all is set, you can make the pull request!
 
 5. Go to ``https://github.com/<your username>/arrow`` where you will see a box with
    the name of the branch that you pushed and next to it a green button
-   **Compare & Pull Request**. After clicking on it, you should add a
-   title and description of the Pull Request. Underneath you can check
+   **Compare & pull request**. After clicking on it, you should add a
+   title and description of the pull request. Underneath you can check
    once again the changes you have made.
 
    .. seealso::
       
-      Get more details on naming the Pull Request in Arrow repository
+      Get more details on naming the pull request in Arrow repository
       and other additional information :ref:`pull_request_and_review`
       section.
 
 Reviews and merge of the pull request
 =====================================
 
 When the pull request is submitted it waits to get reviewed. One of
+<<<<<<< HEAD
+great things about Open Source is your work gets lots of feedback and
+=======
 great things about open source is that your work can get lots of feedback and
+>>>>>>> 2fd43586537d10ebdd65131dd3d0495c1f2b280d
```

Comment:
Lines 132-136 have artifacts of resolving a merge conflict left in them

Reply:
Oops!